react-hot-button-component/src/hot-button.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+let count = 0;
+
```

Comment:
Don't use a variable like this it makes rendering multiple buttons impossible, just add one to the previous state and update the state to the new value.